hack/make.sh

```diff
@@ -157,14 +157,18 @@ main() {
 	else
 		bundles=("$@")
 	fi
-		export DEST="${bundle_dir}/$(basename "$bundle")"
```

Comment:
Ah, deleted in the next commit; that's what I get for reviewing commit-by-commit, I guess. :joy:  That being said, it seems like all the changes in this file were intended to be in that first commit? :thinking:

Reply:
I'll take another pass over the diffs to make sure that I divided things correctly.